packages/next-swc/crates/next-core/src/next_import_map.rs

```diff
@@ -38,6 +39,57 @@ use crate::{
     util::NextRuntime,
 };
 
+const NODE_INTERNALS: [&str; 48] = [
```

Comment:
Could this be composed from https://github.com/vercel/turbo/blob/f714840e331fee4008b85a8d47687dc5c2a163f1/crates/turbopack-resolve/src/resolve.rs#L18-L71?

Reply:
I was debating to leave comment about those - those 2 are actually different. next.js have some global augmentation (like buffer) and there are few apis available in runtime (https://edge-runtime.vercel.app/features/available-apis) .